src/labels/CustomLabels.labels

```diff
@@ -2221,15 +2221,20 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Spinner text when Elevate Commitment is being created</shortDescription>
-        <value>Creating Elevate recurring payment...This may take a few minutes.</value>
+        <value>Saving Elevate recurring payment...This may take a few minutes.</value>
     </labels>
     <labels>
         <fullName>RD2_EntryFormSaveCommitmentFailedMessage</fullName>
         <categories>Recurring-Donations, Entry-Form</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Message displayed as prefix to errors returned from Elevate API</shortDescription>
-        <value>However, an Elevate recurring payment could not be created due to the following errors:</value>
+        <value>A recurring payment has been created in Elevate. Elevate Recurring Id: {0}.
```

Comment:
@sam-knox , can you please review the label as well as PR description? All user profiles regardless if they use Elevate need to have read-only access on specified fields (this is due to using the fields in the form). Please let me know if there are questions.

Reply:
@RadSFDeveloper - Ok, will do. Are there plans to include these new fields in the unmanaged metadata so that at least the Sys Admin profile will get the correct FLS when upgrading to RD2?

Also, are these fields part of the Enhanced Recurring Donation page layout? Would a non-Elevate customer need to remove them?

Reply:
I have updated Admin and Standard Profiles deployed in RD2 meta deploy to include the "Elevate Recurring Id" (`CommitmentId__c`). Other fields were already there.

The Elevate fields are not added to the RD page layout. The admin for the org connected to Elevate needs to add them manually (if required).

Reply:
@sam-knox , to clarify, the specified fields visibility is required only for RD2 customers.

Reply:
@sam-knox , the latest testing on a scratch org and sandbox indicate the fields need to be visible only for RD2 customers using Elevate. Can you please update the PR description? We will also check into sandboxes after the release to confirm.